spark/sql-20/build.gradle

```diff
@@ -38,7 +46,9 @@ compileScala {
         "-Yno-adapted-args",
         "-Ywarn-dead-code",
         "-Ywarn-numeric-widen",
-        "-Xfatal-warnings"
+        "-Xfatal-warnings",
+        "-Xplugin:" + configurations.scalaCompilerPlugin.asPath,
```

Comment:
I see that in the sql-13 build file we add these with `tasks.withType(...)` instead of appending here. I think I'm fine with either, but we should try to do the same in both files.

Reply:
done https://github.com/elastic/elasticsearch-hadoop/pull/1368/commits/2e42e73253e919392b78dfc8bb8c17769ff48d16